zeppelin-server/src/main/java/org/apache/zeppelin/service/ShiroAuthenticationService.java

```diff
@@ -434,7 +434,7 @@ private List<String> getUserList(JdbcRealm obj) {
         return userlist;
       }
 
-      userquery = String.format("SELECT %s FROM %s", username, tablename);
+      userquery = String.format("SELECT ? FROM ?", username, tablename);
```

Comment:
I think that `String.format` is no longer necessary. What do you think?
```suggestion
      userquery = "SELECT ? FROM ?";
```

Reply:
+1. Thanks. Updated.